components/popover/stories/popover.stories.js

```diff
@@ -197,8 +199,169 @@ Nested.args = {
 							label: "Delete",
 						},
 					],
+					...passthroughs,
+				}, context),
+			],
+			...passthroughs,
+		}, context),
+	],
+};
+
+
+// ********* VRT ONLY ********* //
+// TODO: reference PopoverGroup() instead of Template()
+// export const WithForcedColors = PopoverGroup.bind({});
+export const WithForcedColors = Default.bind({});
+WithForcedColors.args = Default.args;
+WithForcedColors.tags = ["!autodocs", "!dev"];
+WithForcedColors.parameters = {
+	chromatic: {
+		forcedColors: "active",
+		// modes: disableDefaultModes
+	},
+};
+
+// ********* DOCS ONLY ********* //
+// TODO: remove any "is-hidden-story" tags once this branch is rebased with main and storybook has been updated to 8.3.*
+/**
+ * Popovers can have a tip. A tip should be used to help show the connection to its source, in cases
+ * where the source does not have a visually distinct down state.
+ */
+export const WithTip = Template.bind({});
+WithTip.storyName = "Default with tip";
+WithTip.tags = ["is-hidden-story", "!dev"];
+WithTip.args = {
+	...Default.args,
+	withTip: true,
+};
+WithTip.parameters = {
+	chromatic: {
+		disableSnapshot: true,
+	},
+};
+
+/**
+ * Popovers can display different elements within their content area. This example uses the
+ * [dialog](?path=/docs/components-dialog--docs) component within the popover.
+ */
+// @see https://opensource.adobe.com/spectrum-web-components/components/popover/#dialog-popovers
+export const DialogStyle = Template.bind({});
```

Comment:
This isn't displaying correctly on the docs page. 

maybe we need to remove `display: inline-flex;` for this story?